tests/gpt_tagger/_integration_test.py

```diff
@@ -0,0 +1,56 @@
+import json
+import logging
+from pprint import pprint
+
+import pytest
+
+from nxontology_ml.efo import get_efo_otar_slim
+from nxontology_ml.gpt_tagger._chat_completion_middleware import (
+    _ChatCompletionMiddleware,
+)
+from nxontology_ml.gpt_tagger._gpt_tagger import GptTagger
+from nxontology_ml.gpt_tagger._models import TaskConfig
+from nxontology_ml.gpt_tagger._openai_models import Response
+from nxontology_ml.gpt_tagger._utils import node_to_str_fn
+from nxontology_ml.utils import ROOT_DIR
+from tests.gpt_tagger._utils import precision_config
+from tests.utils import get_test_nodes, read_test_resource
+
+
+@pytest.mark.skip(reason="IT: Makes a real openai api call")
```

Comment:
At first I thought IT meant Information Technology haha, then realized Integration Test. Might eventually create a `pytest.mark.IT` that would automatically skip everything with that mark. No need to do this now.

Reply:
Yes, that's what we did in my previous company.

Regarding `IT`: Which name convention do you guys use internally? Do you want me to spell it out?

Reply:
No problem, keep as is. Just really a note to self there.